modules/apps/transfer/keeper/migrations_test.go

```diff
@@ -235,3 +236,82 @@ func (suite *KeeperTestSuite) TestMigrateTotalEscrowForDenom() {
 		})
 	}
 }
+
+func (suite *KeeperTestSuite) TestMigratorMigrateMetadata() {
+	var (
+		denomTrace       transfertypes.DenomTrace
+		expectedMetadata banktypes.Metadata
+	)
+
+	testCases := []struct {
+		msg      string
+		malleate func()
+	}{
+		{
+			"success with denom trace with one hop",
```

Comment:
needs test cases for:
- multiple denom traces
- 1 already set denom trace with metadata and 1 without metadata set

Reply:
Added the tests.